### DISCUSSION TITLE
Page loading suddenly super slow

### DISCUSSION THREAD

**Discussion by tensorFan** [alshedivat/al-folio#372] [Q&A] [answered]
[Here](https://tensorfan.github.io/) is my Jekyll site, and here is its associated [github](https://github.com/tensorFan/tensorFan.github.io/tree/source) page. The site is forked from the theme al-folio and used to load really quickly without any issues. After I came back from vacation it takes >60s to load each page, and I have no idea why. I should mention that I know basically nothing about webdev so I would greatly appreciate any help.

**Comment by JulianStreyczek** [comment, marked as answer]
Hi! You can try changing "https://gitcdn.xyz/..." to "https://gitcdn.link/..." in this line in your repository:

https://github.com/tensorFan/tensorFan.github.io/blob/355951fa9a634c70cc8794ef87b80663987d372b/_includes/head.html#L27-L28

The line loads a [CSS style sheet](https://www.w3schools.com/html/html_css.asp) from another repository on Github, but it seems like one of the two addresses may be significantly slower depending on your location.

In the main project, the .xyz link was an attempt to improve loading times (see #234), but was reverted to .link recently (see #364). The topic is still an open issue as of now (#367).

**Reply by tensorFan** [reply]
That was it, thank you so much! Hopefully a more stable link gets found in the future. I can't mark your comment as answer? Clicking the button doesn't do anything

**Reply by JulianStreyczek** [reply]
Weird. But glad I could help!

**Reply by JacobFV** [reply]
Thank you. Thank you. Thank you

**Reply by soham97** [reply]
Thanks, changing from `.xyz` to `.link` does solve the issue!

**Reply by ShengranHu** [reply]
thanks! That helps me a lot.

**Reply by magusd** [reply]
I ended up on this page because I had a github pages publishing issue, the base_url was pointing somewhere else so all the assets requests were being sent to invalid links. This solved the problem for me, thanks a lot!

**Reply by tensorFan** [reply]
I am way behind the main repo now on my fork, but it got slow again today. If anyone else is on an older version it seems that following NeoNeuron's advice in this thread https://github.com/alshedivat/al-folio/issues/367, changing the same line to https://combinatronics.io/jwarby/pygments-css/master/github.css, seems to work to speed things back up again.

**Reply by HaoranLiao** [reply]
Mine is also way behind the main repo got really slow recently! Tried @tensorFan's latest reply, changing the same line to https://combinatronics.io/jwarby/pygments-css/master/github.css worked!

**Reply by leo-liuzy** [reply]
Mine (**https://gitcdn.link/..**) is super slow; and change that line in `_includes/head.html` to `https://combinatronics.io/jwarby/pygments-css/master/github.css` really works!
